nni/retiarii/evaluator/pytorch/lightning.py

```diff
@@ -22,11 +22,11 @@ def set_model(self, model):
             self.model = model
 
 
-Trainer = blackbox_module(pl.Trainer)
-DataLoader = blackbox_module(DataLoader)
+Trainer = serialize_cls(pl.Trainer)
+DataLoader = serialize_cls(DataLoader)
 
 
-class Lightning(TrainingConfig):
+class Lightning(ModelEvaluator):
```

Comment:
what is the difference between `ModelEvaluator`, `Evaluator` and `Trainer`?

Reply:
Maybe we can deprecate the concept of trainer and rename DartsTrainer -> DARTS, EnasTrainer -> ENAS? Leave that for discussion.